pkg/framework/aries/defaults/defaults.go

```diff
@@ -27,9 +27,15 @@ func WithInboundHTTPAddr(internalAddr, externalAddr, certFile, keyFile string) a
 }
 
 // WithInboundWSAddr return new default ws inbound transport.
-func WithInboundWSAddr(internalAddr, externalAddr, certFile, keyFile string) aries.Option {
+func WithInboundWSAddr(internalAddr, externalAddr, certFile, keyFile string, readLimit int64) aries.Option {
```

Comment:
it would be better to add a comment here about the default value when readLimit is 0. Or may be we can add another option function with readLimit.